include/crow/websocket.h

```diff
@@ -620,6 +628,7 @@ namespace crow
             WebSocketReadState state_{WebSocketReadState::MiniHeader};
             uint16_t remaining_length16_{0};
             uint64_t remaining_length_{0};
+            uint64_t max_payload_bytes_{UINT64_MAX};
```

Comment:
No idea what the best default value is here so I put in the max for payload in WebSocket payload length

Reply:
Yeah I don't think 18.4 Exabytes is a useful value in this case 😅😆

Reply:
should I put it as uint32_max? That's 4 gigabytes.

Reply:
I'll have to look into it a bit more. My current opinion is to disable the feature by default and allow developers to enable it if they want to. but that would require #422 to be implemented first.

Reply:
I actually have a version where in the WebSocketRule you can call a public function payloadmax() but I saw the #422 dependency so decided to just commit this rudimentary version. I also agree maybe to not break anything for now we can have the UINT64_MAX to disable the feature.

Reply:
I never actually thought of that, it allows setting the max value per route. Maybe a hybrid of the 2 where you define a global max and can override per route would be the best of both worlds..

Reply:
I only went with the WebSocketRule version because I didn't understand what was required to set it globally per app. And the websocket() function seemed like a good place to start looking at how I could customize my route. I'm willing to add my whole version in this pull request, but I don't know how I can do the global per-app solution.